site/content/docs/main/_index.md

```diff
@@ -1,7 +1,6 @@
 ---
 cascade:
   layout: docs
-  gh: https://github.com/projectcontour/contour/tree/release-1.15.1
```

Comment:
AFAICT this isn't used anywhere (this isn't actually a valid link anyway), so I removed it so it doesn't have to be updated as part of `prepare-release.go`. LMK if you're aware of a need for this!